### ISSUE TITLE
Lag in editor object?

### ISSUE THREAD

**Issue by techies-sudo** [jerosoler/Drawflow#151]
Say a node has 3 outnodes, if all three nodes are connected to separate nodes
![Screenshot from 2021-04-03 16-35-58](https://user-images.githubusercontent.com/61532920/113476741-fc145f00-949a-11eb-98c6-b08493647e34.png)

if I am removing one node say first one(class which is in right side of the node)  ,  3 will be removed instead 1 by the library.
console.log(editor.drawflow.drawflow.Home.data,state.editor.getNodeFromId(id)) 
![Screenshot from 2021-04-03 17-03-58](https://user-images.githubusercontent.com/61532920/113477325-9e821180-949e-11eb-82db-37af303ae2e8.png)
though both are same object, outputs object varies , why is this?

**Comment by jerosoler** [comment]
Hello,

Who wants to delete the connection?
or the Output?

Have you tried events?
https://github.com/jerosoler/Drawflow#events

Jero

**Comment by techies-sudo** [comment]
So there is a situation that I want to delete a class, here o deleted class 1 , I used editor,removeNodeOutput(id, output_class), which deletes class, so connection removal event triggered, then I console both editor and getNodeFromId(id). as I shown above, outputs object differs even though it's the same node . output_1 in right side shows empty object which makes sense since there is nothing connected to it. but it I deleted the object, later I figured that connections to output_2 shifted to output_1 and output_3 to output_2 as shown in left side picture. Why is it?
